packages/user-interface/src/pages/TaskPage.tsx

```diff
@@ -34,48 +35,65 @@ const TaskPage = ({ backlink = {} }: TaskPageProps) => {
   const [submission, setSubmission] = useState({
     data: {},
   });
+  const [taakVersion, setTaakVersion] = useState(TaakVersion.V2);
 
-  const [submitTask] = useSubmitTaskMutation({
+  const [submitTaak] = useSubmitTaakV2Mutation({
     onCompleted: () => {
       setSubmitted(true);
       client.cache.reset();
```

Comment:
Weet dat je hem niet hebt aangepast. Maar kunnen we die iets netter maken door alleen te updaten wat aangepast is? zie voorbeeld: https://github.com/nl-portal/nl-portal-frontend-libraries/blob/master/packages/user-interface/src/pages/EditAccountPage.tsx